go.mod

```diff
@@ -29,5 +29,5 @@ require (
 	k8s.io/apimachinery v0.20.1
 	k8s.io/client-go v0.20.1
 	sigs.k8s.io/controller-runtime v0.8.0
-	sigs.k8s.io/controller-tools v0.3.0
```

Comment:
@muvaf was bumping version here intentional?

Reply:
Yes, `make reviewable` made some changes I think not compatible with clean CRD target. So, I just bumped it to make it inline with other providers.

Reply:
Thanks!